auto_round/autoround.py

```diff
@@ -2331,7 +2331,7 @@ def quantize_block(self, block, input_ids, input_others, q_input=None, device=to
         """
         if hasattr(self.model, "is_fp8"):
             for n, m in block.named_modules():
-                if m.__class__.__name__ == "FP8Linear":
+                if hasattr(m, "is_fp8"):
```

Comment:
why make this change, convert_fp8_layer_to_linear could not skip non-linear layer